aws/src/integration/java/org/apache/iceberg/aws/dynamodb/TestDynamoDbCatalog.java

```diff
@@ -379,6 +379,21 @@ public void testRegisterTable() {
     Assertions.assertThat(catalog.dropNamespace(namespace)).isTrue();
   }
 
+  @Test
+  public void testDefaultWarehousePathWithLocation() {
```

Comment:
Thank you for your contribution. I was wondering if it would be possible to add a unit test for this change in addition to the aws integration test. like:
https://github.com/apache/iceberg/blob/b78d3361635e924ed23fd8fe87b5f966b1953dd9/aws/src/test/java/org/apache/iceberg/aws/glue/TestGlueCatalog.java#L143-L153
Since the AWS integration test is not currently run with CI, having a unit test could help catch errors more quickly and ensure that future development reflects this change accurately. What do you think?

Reply:
+1 for adding a unit test, @munendrasn let me know if you could add that as suggested, other things looks good to me.

Reply:
I have included the unit tests, but for now, included tests only for defaultwarehouse location path in unit tests